testsuite/factories.py

```diff
@@ -110,6 +110,38 @@ class Meta:
                    'manage_media_participant', 'view_participant')
 
 
+class RoleUserMeetingFactory(SQLAlchemyModelFactory):
+
+    class Meta:
+        model = models.RoleUser
+        sqlalchemy_session = models.db.session
+
+    role = SubFactory(RoleFactory)
+    user = SubFactory(UserFactory)
+    meeting = SubFactory(MeetingFactory)
+
+    @post_generation
+    def staff(self, create, extracted, **kwargs):
+        if not create:
+            return
+        staff = extracted or StaffFactory(user=self.user)
+        return staff
+
+
+class JobFactory(SQLAlchemyModelFactory):
+    class Meta:
+        model = models.Job
+        sqlalchemy_session = models.db.session
+
+    id = Sequence(lambda n: n)
+    name = 'short list'
+    user_id = Sequence(lambda n: n)
```

Comment:
Use a [SubFactory](http://factoryboy.readthedocs.io/en/latest/recipes.html#dependent-objects-foreignkey) for related objects. Fields with _UNIQUE_ constraint can be easily initialized with default values by using a [Sequence](http://factoryboy.readthedocs.io/en/latest/recipes.html#forcing-the-sequence-counter).